app/src/components/v-workspace-tile.vue

```diff
@@ -72,7 +72,7 @@
 			<div class="bottom-left" @pointerdown.stop="onPointerDown('resize-bottom-left', $event)" />
 		</div>
 
-		<div class="tile-content" :class="{ 'has-header': showHeader }">
+		<div v-if="!loading" class="tile-content" :class="{ 'has-header': showHeader }">
```

Comment:
I'm not sure if we should have the `loading` state prop on the workspace tile abstraction, or if that's the responsibility of the panel component that relies on this parent component. The tile abstraction itself doesn't care about what it displays, and therefore doesn't really have to care about loading states of that content either